roles/etcd/templates/etcd.j2

```diff
@@ -12,6 +12,9 @@
   {% if etcd_cpu_limit is defined %}
   --cpu-shares={{ etcd_cpu_limit|regex_replace('m', '') }} \
   {% endif %}
+  {% if etcd_blkio_weight is defined %}
```

Comment:
The if is not really necessary since you set a default, but I will merge it anyway.